src/fields/jsgrid.field.select.js

```diff
@@ -100,13 +100,15 @@
 
             $.each(this.items, function(index, item) {
                 var value = valueField ? item[valueField] : index,
-                    text = textField ? item[textField] : item;
+                    text = textField ? item[textField] : item,
+                    disabled = item['disabled'];
```

Comment:
we use double-quotes usually, let's be consistent. While I'd rather just use dot notation here:
`disabled = (item.disabled === true)`.